src/SdkGenerator/SdkGenerator.csproj

```diff
@@ -1,74 +1,85 @@
+        Version="1.1.1"
+        PrivateAssets="all" />
+  </ItemGroup>
 
-    <ItemGroup>
-        <None Update="Templates\csharp\ApiClient.cs.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\csharp\sdk.nuspec.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\csharp\ApiInterface.cs.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\java\ApiClient.java.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\python\ApiClient.py.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\python\__init__.py.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\ruby\ApiClient.rb.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\ts\ApiClient.ts.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\ts\index.ts.scriban">
-            <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\dart\ApiClient.dart.scriban">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\dart\ApiInterface.dart.scriban">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\java\RestRequest.java.scriban">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\java\pom.xml.scriban">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\python\setup.cfg.scriban">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\ts\package.json.scriban">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="Templates\csharp\nuget-publish.yml.scriban">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-        <None Update="DotnetToolSettings.xml">
-          <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-        </None>
-    </ItemGroup>
+  <ItemGroup>
+    <None
+        Include="README.md"
+        PackagePath="/docs"
+        Pack="true" />
+    <None
+        Include="icons-puzzle.png"
+        PackagePath="/docs"
+        Pack="true" />
+    <None
+        Include="PatchNotes.md"
+        PackagePath="/docs"
+        Pack="true" />
+    <None
+        Include="Templates/**/*"
+        Pack="true"
+        CopyToPublishDirectory="PreserveNewest" />
```

Comment:
You can include all kinds of build content in a package using the `Pack="true"` metadata. If you do, you can use `PackagePath=""` to specify a directory it should be in _inside the package_. Finally, most kinds of build items support the `CopyToPublishDirectory` flag (which is also used during packaging) separately to `CopyToOutputDirectory` (which thinking about it now I guess you'd still want here). I would very much suggest using `PreserveNewest` in all cases those, as `Always` prevents your build from being incremental.